modules/core/api.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Change the license header. Check other places as well

Reply:
Fixed with commit `5015480`.